h2o-algos/src/main/java/hex/tree/isoforextended/ExtendedIsolationForestMojoWriter.java

```diff
@@ -0,0 +1,33 @@
+package hex.tree.isoforextended;
+
+import hex.tree.SharedTreeMojoWriter;
+import hex.tree.isofor.IsolationForestModel;
+
+import java.io.IOException;
+
+import static hex.tree.isofor.IsolationForestModel.IsolationForestOutput;
+import static hex.tree.isofor.IsolationForestModel.IsolationForestParameters;
+
+/**
+ * Mojo definition for Extended Isolation Forest model.
+ */
+public class ExtendedIsolationForestMojoWriter extends SharedTreeMojoWriter<ExtendedIsolationForestModel, ExtendedIsolationForestModel.ExtendedIsolationForestParameters, ExtendedIsolationForestModel.ExtendedIsolationForestOutput> {
```

Comment:
no reason to commit this NOOP class until MOJO is really implemented

Reply:
Implemented in 1a9d25b2d41ce5d77d8a84e221adbba5a5c23e3a on PR: https://github.com/h2oai/h2o-3/pull/4907